tests/integration/docusaurus/connecting_to_your_data/cloud/azure/pandas/configured_python_example.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+from ruamel import yaml
+
+import great_expectations as ge
+from great_expectations.core.batch import BatchRequest, RuntimeBatchRequest
+
+CREDENTIAL = os.getenv("AZURE_CREDENTIAL", "")
+
+context = ge.get_context()
+
+datasource_config = {
+    "name": "my_azure_datasource",
+    "class_name": "Datasource",
+    "execution_engine": {
+        "class_name": "PandasExecutionEngine",
+        "azure_options": {
+            "account_url": "<YOUR_ACCOUNT_URL>",
+            "credential": "<YOUR_CREDENTIAL>",
+        },
+    },
+    "data_connectors": {
+        "configured_data_connector_name": {
+            "class_name": "ConfiguredAssetAzureDataConnector",
+            "azure_options": {
+                "account_url": "<YOUR_ACCOUNT_URL>",
+                "credential": "<YOUR_CREDENTIAL>",
+            },
```

Comment:
We will reflect the nuances between `inferred` and `configured` in the docs but for purposes of this ticket, I advocate that we just focus on the quality of the test

Reply:
sure thing